### PR TITLE
When an issue is opened, label it with the "Needs actionability review" label.

### PR DESCRIPTION
**Pull request by jverkoey**
Closes https://github.com/material-foundation/material-automation/issues/10

### CODE REVIEW

File: Sources/main.swift [hunk 1]
```diff
@@ -102,6 +102,11 @@ routes.add(method: .post, uri: "/webhook", handler: { request, response in
       if let titleLabel = PRLabelAnalysis.getTitleLabel(title: issueData.title) {
         labelsToAdd.append(titleLabel)
       }
+      let actionabilityLabel = "Needs actionability review"
```
Comment (jverkoey) [inline Sources/main.swift]: Should I move this to `ProcessInfo.processInfo.environment`?

Comment (jverkoey) [reply Sources/main.swift]: If so, where do I define it?

Comment (yarneo) [reply Sources/main.swift]: yeah, we should be able to define it but currently can't. I have opened an issue for it here: #18

File: Sources/main.swift [hunk 2]
```diff
@@ -102,6 +102,11 @@ routes.add(method: .post, uri: "/webhook", handler: { request, response in
...
+      if githubData?.action == "opened"
+          && !issueData.labels.contains(where: { $0 == actionabilityLabel }) {
```
Comment (yarneo) [inline Sources/main.swift]: swift continuation lines are traditionally 2 spaces (not 4)

**Review by yarneo — APPROVED** [review]
Please rebase and fix conflict, otherwise LGTM!

**Comment by jverkoey** [comment]
Rebased. Please take a look again because the code pattern has changed.

File: Sources/LabelAnalysis.swift [hunk 3]
```diff
@@ -242,4 +242,11 @@ class LabelAnalysis {
     return distance[str1.count][str2.count] <= threshold
   }
 
+  /// Adds a "Needs actionability review" label to the issue.
+  class func addNeedsActionabilityReviewLabel(issueData: IssueData) {
+    let actionabilityLabel = "Needs actionability review"
+    if !issueData.labels.contains(where: { $0 == actionabilityLabel }) {
+      GithubAPI.addLabelsToIssue(url: issueData.url, labels: Array(Set(labelsToAdd)))
```
Comment (yarneo) [inline Sources/LabelAnalysis.swift]: Array(Set(labelsToAdd)) -> [actionabilityLabel]

**Review by yarneo — DISMISSED** [review]

File: Sources/LabelAnalysis.swift (hunk 3 above)
Comment (jverkoey) [reply Sources/LabelAnalysis.swift]: Done.

**Review by yarneo — APPROVED** [review]

**Pull request merged**